sites/public/src/components/listings/ListingsList.tsx

```diff
@@ -1,12 +1,39 @@
 import * as React from "react"
 import { getListings } from "../../lib/helpers"
 import { Listing } from "@bloom-housing/backend-core"
+import { Button, InfoCard, t } from "@bloom-housing/doorway-ui-components"
 
 type ListingsListProps = {
   listings: Listing[]
 }
 
 const ListingsList = (props: ListingsListProps) => {
-  return <div className="listingsList">{getListings(props.listings)}</div>
+  return (
+    <div>
+      <div className="listingsList">{getListings(props.listings)}</div>
+      {/* TODO: once pagination is implemented for listings, the following should only show on the last page. */}
+      <InfoCard
+        title={t("t.signUpForAlerts")}
+        subtitle={t("t.subscribeToNewsletter")}
+        className="is-normal-primary-lighter"
+      >
+        <Button className="is-primary">{t("t.signUp")}</Button>
+      </InfoCard>
+      <InfoCard
+        title={t("t.needHelp")}
+        subtitle={t("t.emergencyShelter")}
+        className="is-normal-secondary-lighter"
+      >
+        <Button className="is-secondary">{t("t.helpCenter")}</Button>
+      </InfoCard>
+      <InfoCard
+        title={t("t.housingInSanFrancisco")}
+        subtitle={t("t.seeSanFranciscoListings")}
+        className="is-normal-secondary-lighter"
+      >
+        <Button className="is-secondary">{t("t.seeListings")}</Button>
```

Comment:
@ifranch if you want to link to DAHLIA, here's the site `https://housing.sfgov.org/`  

we'll soon have the listings alert URL set up, but that one is blocked for now.

Reply:
Done, I made it open in a new tab as well. The component changes from Button to LinkButton but it uses the same styling as Button so no additional styling changes are needed.